pennsignals/pennsignals.github.io
# Add Post/data challenges in health care

A post setting up the series on healthcare data challenges and PennMed DS solutions.

This is a WIP -- please comment and edit liberally.

Review instructions:
- Read text
- Check for spelling and grammatical errors
- Ensure that the content adequately sets up your planned post in this series
